fhir-server/liberty-config/server.xml

```diff
@@ -99,6 +99,12 @@
     </dataSource>
 
     <webAppSecurity allowFailOverToBasicAuth="true" singleSignonEnabled="false"/>
+    
+    <cors domain="/"
+        allowedOrigins="*"
```

Comment:
I don't think this is a best practice.  
We should probably limit to localhost or externalize the variable.

Reply:
I've heard this before too, although I'm not familiar with the downsides of allowing all origins.
The spec specifically calls out `*` as the "easiest option", although it does stop short of saying that it is recommended / not recommended.
https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.security.cors

Reply:
interesting... https://advancedweb.hu/is-access-control-allow-origin-star-insecure/ it's not all that bad from this article